app/cafe.py

```diff
@@ -0,0 +1,21 @@
+import datetime
+from app.errors import NotWearingMaskError
+from app.errors import OutdatedVaccineError
+from app.errors import NotVaccinatedError
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("Not vaccinated")
+        if visitor["vaccine"]["expiration_date"] < datetime.date.today():
+            raise OutdatedVaccineError("vaccination outdated")
+        if (
+                "wearing_a_mask" not in visitor
+                or visitor["wearing_a_mask"] is False
+        ):
+            raise NotWearingMaskError("Not wear mask")
+        return f"Welcome to {self.name}"

```

Comment:
Also i recommend you to use visitor's name in your error messages to make them more descriptive